lib/rubocop/cop/rails/save_bang.rb

```diff
@@ -35,6 +35,26 @@ module Rails
       #   unless user.persisted?
       #     # ...
       #   end
+      #
+      # @example AllowImplicitReturn: true (default)
```

Comment:
I think the cop description should be extended to account for this configurable behaviour.